src/commands/ext-update.ts

```diff
@@ -62,10 +65,96 @@ export default new Command("ext:update <extensionInstanceId> [updateSource]")
   .before(diagnoseAndFixProject)
   .withForce()
   .option("--params <paramsFile>", "name of params variables file with .env format.")
-  .action(async (instanceId: string, updateSource: string, options: any) => {
+  .option(
+    "--local",
+    "save the update to firebase.json rather than directly update at a Firebase project"
```

Comment:
Nit: consider rewording this to "rather than directly update an existing Extension instance on a Firebase project"